File: app/src/main/java/com/example/ibrahim/servisinfo_ib150071/data/GradoviResultVM.java
package com.example.ibrahim.servisinfo_ib150071.data;

import java.io.Serializable;
import java.util.List;

/**
 * Created by Ibrahim on 27.7.2018..
 */

public class GradoviResultVM implements Serializable {
    public static class Row implements Serializable
    {
        public int GradID ;
        public String Naziv ;
    }

    public List<GradoviResultVM.Row> rows;


}

File: RKS_IB150071_WebServisi/RKS_IB150071_WebServisi/Model2/GradoviResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RKS_IB150071_WebServisi.Model2
{
    public class GradoviResultVM
    {

        public class Row
        {
            public int GradID { get; set; }
            public string Naziv { get; set; }
        }

        public List<Row> rows;
    }
}